contracts/utils/Errors.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+/**
+ * @dev Collection of standard custom error that are used in multiple contracts
+ */
+library Errors {
+    /**
+     * @dev The ETH balance of the account is not enough to perform the operation.
+     */
+    error InsufficientBalance(uint256 balance, uint256 needed);
+
+    /**
+     * @dev A call to an address target failed. The target may have reverted.
+     */
+    error FailedInnerCall();
```

Comment:
`FailedCall`, `FailedInnerCall` and `FailedSubCall` are all good to me. My favorite is `FailedCall`, because simplicity ... but all are good honestly.

Reply:
Note that this error is also emitted when the call that fails is actually a static call or a delegate call

Reply:
> I also like the symetry of FailedCall <> FailedDeployment

Let's do FailedCall, I think it covers the static and delegate call as well.